src/app/HorizontalNav/HorizontalNav.tsx

```diff
@@ -6,14 +6,21 @@ import Avatar_image from '!!url-loader!@app/Assets/images/Avatar_image.svg';
 import CheckCircleIcon from '@patternfly/react-icons/dist/esm/icons/check-circle-icon';
 import ExclamationCircleIcon from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
 import nodeContext from '@app/ContextStore/nodeContext';
+import {getHostname, getPort} from "@app/CentralConfig";
 
 const HorizontalNav = () => {
   const [activeItem, setActiveItem] = useState(0);
   const [autotuneOn, setAutotuneOn] = useState(false);
   const [clusterStatus, setClusterStatus] = useState(false);
-  const Context = useContext(nodeContext);
-  const ip = Context['cluster'];
-  const port = Context['autotune'];
+  // const Context = useContext(nodeContext);
+  // const ip = Context['cluster'];
+  // const port = Context['autotune'];
+
+  console.log("hello");
```

Comment:
can we remove this console log statement as it just prints hello

Reply:
Removed